.github/workflows/lint.yml

```
@@ -12,3 +12,12 @@ jobs:
         node-version: 16
     - run: npm i
     - run: npm run lint
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
```

v2 also allows native caching https://github.com/actions/setup-node#caching-packages-dependencies if you set it. caching probably isn't needed right now, since there are no lock files
```suggestion
    - uses: actions/setup-node@v2
```